runsc/boot/mount_hints.go

```diff
@@ -252,6 +255,15 @@ func (r *RootfsHint) setType(val string) error {
 	return nil
 }
 
+func (r *RootfsHint) setOptions(val string) error {
+	size, cut := strings.CutPrefix(val, "size=")
```

Comment:
Should we first split `val` by `,` and then process the result? To make room for multiple options in the future. That would also be consistent with how `--overlay2` flag is processed; its value is split by "," and we ensure that only 1 option is present, if any and that option should be size.